src/styles/main.scss

```diff
@@ -1,7 +1,13 @@
-@import "utils";
-@import "fonts";
-@import "typography";
-
-body {
-  background: $c-gray;
-}
+@import "../styles/utils";
+@import "../styles/header.scss";
```

Comment:
The 'fonts' and 'typography' imports are missing from your main.scss file. If these files contain global styles or font-face declarations, it's important to include them. You might want to add '@import "fonts";' and '@import "typography";' back to your code.